README.md

```diff
@@ -1,136 +1,135 @@
 
 It will use the ansible playbook and roles in this repository to perform the VM provisioning. However,
 Vagrant generates it's own host inventory since it operates using the vagrant user. eg.
 
-```yaml
+```ruby
 ANSIBLE_GROUPS = {
   "all_groups" => ["rockstor"],
   "all_groups:vars" => {
         "rs_inst_set_root_pass" => "true"
-        }
+    }
 }
 ```
 
-Pre-requesits
--------------
+## Prerequisites
+
 It is assumped that you have the follow software packages install on you host machine:
-- Vagrant (https://www.vagrantup.com/downloads)
+
+- [Vagrant](https://www.vagrantup.com/downloads)
 - Ansible - see this [README](ansible/README.md) for download instructions.
 
-All above packages are available on Linux, Mac OSX and Windows.
+All above packages are available on Linux, MacOS, and Windows.
 
 You will also need one of the following virtualisation platforms:
-  - <b>Linux users:</b> may prefer to use libvirt, qemu + kxm (see https://github.com/vagrant-libvirt/vagrant-libvirt)
 
-    or 
-  - Virtualbox is available on all platforms (https://www.virtualbox.org/wiki/Downloads)
- 
+- **Linux users:** may prefer to use libvirt, qemu + kvm (see [vagrant-libvirt](https://github.com/vagrant-libvirt/vagrant-libvirt))
+- Virtualbox is available on all platforms: [Download](https://www.virtualbox.org/wiki/Downloads)
 
-It is also assumed that you have a local copy of the core repository [rockstor-core](https://github.com/rockstor/rockstor-core) 
+It is also assumed that you have a local copy of the core repository [rockstor-core](https://github.com/rockstor/rockstor-core)
 on your machine at the same level as this repo:
 
-```
-.../rockstor-core
-.../rockstor-dev-env
-
+```sh
+./rockstor-core
+./rockstor-dev-env
 ```
 
-You should also be currently within the 'vagrant_env' directory in this repo, where the 'Vagrantfile' is located.
+You should also be currently within the [vagrant_env](/vagrant_env) directory in this repo, where the `Vagrantfile` is located.
 
-Vagrant Boxes
--------------
+## Vagrant Boxes
 
-This Vagrantfile uses a number of vagrant box: 
+This Vagrantfile uses a number of vagrant boxen:
 
 - [opensuse/leap-15.2.x86_64](https://app.vagrantup.com/opensuse/boxes/Leap-15.2.x86_64)
 - [opensuse/Tumbleweed.x86_64](https://app.vagrantup.com/opensuse/boxes/Tumbleweed.x86_64)
 - [opensuse/Tumbleweed.aarch64](https://app.vagrantup.com/opensuse/boxes/Tumbleweed.aarch64)
 
-Building an OpenSUSE Leap VM and installing Rockstor from source
-----------------------------------------------------------------
-On Mac OSX, Linux and Windows:
+## Building an OpenSUSE Leap VM and installing Rockstor from source
 
-```shell script
+On MacOS, Linux and Windows:
+
+```sh
 ./build.sh
 ```
 
-This will, by default, build and provision an OpenSUSE Leap 15.2 vagrant box in libvirt - including rsyncing the source to the 
-VM in /root. It will then build and install Rockstor from source as per the install guide in the top level README.md.
```

Comment:
I couldn't locate this README, so I removed this phrase, but if we know where it is, we should link to it.

Reply:
Agreed.